src/components/v2/V2Project/V2FundingCycleSection/ReservedTokensSplitsCard.tsx

```diff
@@ -138,21 +143,41 @@ export default function ReservedTokensSplitsCard({
         )}
 
         <div>
-          <TooltipLabel
-            label={
-              <h4 style={{ display: 'inline-block' }}>
-                <Trans>Reserved {tokensText}</Trans> (
-                {formatReservedRate(reservedRate)}%)
-              </h4>
-            }
-            tip={
-              <Trans>
-                A project can reserve a percentage of tokens minted from every
-                payment it receives. Reserved tokens can be distributed
-                according to the allocation below at any time.
-              </Trans>
-            }
-          />
+          <div
+            style={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              gap: 10,
+              flexWrap: 'wrap',
+            }}
+          >
+            <TooltipLabel
+              label={
+                <h4 style={{ display: 'inline-block' }}>
+                  <Trans>Reserved {tokensText}</Trans> (
+                  {formatReservedRate(reservedRate)}%)
+                </h4>
+              }
+              tip={
+                <Trans>
+                  A project can reserve a percentage of tokens minted from every
+                  payment it receives. Reserved tokens can be distributed
+                  according to the allocation below at any time.
+                </Trans>
+              }
+            />
+            {reservedRate?.gt(0) ? (
+              <Button
```

Comment:
**suggestion**: could we add `1rem` or so of bottom margin to improve spacing?

| before | after |
| --- | --- |
| <img width="522" alt="Screen Shot 2022-06-22 at 10 00 25 PM" src="https://user-images.githubusercontent.com/94939382/175029369-45501165-d05f-4ef9-b51a-5f0a23d27d24.png"> | <img width="527" alt="Screen Shot 2022-06-22 at 10 00 04 PM" src="https://user-images.githubusercontent.com/94939382/175029391-fae5a6f7-4c2c-47ea-bfdf-9d071f111400.png"> |